src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -493,6 +493,12 @@
   <data name="PowerLauncher_ClearInputOnLaunch.Content" xml:space="preserve">
     <value>Clear the previous query on launch</value>
   </data>
+  <data name="PowerLauncher_TabSelectsContextButtons.Header" xml:space="preserve">
+    <value>Tab through context buttons</value>
+  </data>
+  <data name="PowerLauncher_TabSelectsContextButtons.Description" xml:space="preserve">
+    <value>Pressing tab will first select through the available context buttons of the current selection before moving onto the next result</value>
```

Comment:
Do we need a not what keys are used without having the setting enabled?

Reply:
> Do we need a not what keys are used without having the setting enabled?

Sorry I did not understand what you mean by this

Reply:
The question is: What key is used without that setting enabled? And should we add the info into the description.

Reply:
I guess the down arrow/up arrow serve this purpose with or without the setting enabled. As the setting only affects the tab behaviour, I don't think adding this information will be useful?